cfgmgr/vlanmgr.cpp

```diff
@@ -341,7 +341,7 @@ void VlanMgr::processUntaggedVlanMembers(string vlan, const string &members)
         SWSS_LOG_ERROR("Failed to find tableName:%s", CFG_VLAN_MEMBER_TABLE_NAME);
         return;
     }
-    Consumer& consumer = consumer_it->second;
+    auto& consumer = (Consumer &)*consumer_it->second;
```

Comment:
original 
consumer_it->second :: Consumer
now
consumer_it->second :: Executor*

The cast is valid, since there is only Consumers in this application. I can change it to static_cast.

---
In reply to: [150726106](https://github.com/Azure/sonic-swss/pull/382#discussion_r150726106) [](ancestors = 150726106)